chalice/awsclient.py

```diff
@@ -104,10 +109,16 @@ def delete_function(self, function_name):
         except lambda_client.exceptions.ResourceNotFoundException:
             raise ResourceDoesNotExistError(function_name)
 
-    def update_function(self, function_name, zip_contents,
-                        environment_variables=None,
-                        runtime=None, tags=None):
-        # type: (str, str, _STR_MAP, _OPT_STR, _STR_MAP) -> Dict[str, Any]
+    def update_function(self,
+                        function_name,  # type: str
```

Comment:
Sure. Is there possibly a linter for this? I have not used type hints too much so I was not aware of that portion of the PEP.